src/main/java/me/zeroeightsix/kami/module/modules/misc/HighwayTools.kt

```diff
@@ -0,0 +1,738 @@
+                    if (blockAction.getBlock()::class == material::class) {
+                        val block = mc.world.getBlockState(blockAction.getBlockPos()).block
+                        if (block !is BlockAir) {
+                            blockAction.setTaskState(TaskState.DONE)
+                            blockAction.setPriority(4)
+                        } else {
+                            blockAction.setTaskState(TaskState.PLACE)
+                        }
+                    } else {
+                        blockAction.setTaskState(TaskState.BREAK)
+                        blockAction.setPriority(2)
+                    }
+                    doTask()
+                } else if (blockAction.getTaskState() == TaskState.DONE) {
+                    blockQueue.poll()
+                    doneQueue.add(blockAction)
+                    doTask()
+                }
+            } else {
+                waitTicks--
+            }
+            return true
+        } else {
+            return false
+        }
+    }
+
+    private fun updateTasks() {
+        updateBlockArray()
+        for ((a, b) in blockOffsets) {
+            val block = mc.world.getBlockState(a).block
+            if (!b && block in ignoreBlocks) { addTask(a, TaskState.DONE, getBlockById(0), 4) }
+            else if (b && block is BlockAir) { addTask(a, TaskState.PLACE, material, 3) }
+            else if (b && block !is BlockAir && block::class != material::class) { addTask(a, TaskState.BREAK, material, 2) }
+            else if (!b && block !is BlockAir) { addTask(a, TaskState.BREAK, getBlockById(0), 2) }
+            else if (b && block::class == material::class) { addTask(a, TaskState.DONE, material, 4) }
+            else if (!b && block is BlockAir) { addTask(a, TaskState.DONE, getBlockById(0), 4) }
+        }
+    }
+
+    private fun mineBlock(pos: BlockPos, pre: Boolean) {
+        if (InventoryUtils.getSlotsHotbar(278) == null && InventoryUtils.getSlotsNoHotbar(278) != null) {
+            InventoryUtils.moveToHotbar(278, 130, (tickDelay.value * 16).toLong())
+            return
+        } else if (InventoryUtils.getSlots(0, 35, 278) == null) {
+            MessageSendHelper.sendChatMessage("$chatName No Pickaxe was found in inventory")
+            mc.getSoundHandler().playSound(PositionedSoundRecord.getRecord(SoundEvents.ENTITY_EXPERIENCE_ORB_PICKUP, 1.0f, 1.0f))
+            disable()
+            return
+        }
+        InventoryUtils.swapSlotToItem(278)
+        lookAtBlock(pos)
+
+        if (pre) {
+            mc.connection!!.sendPacket(CPacketPlayerDigging(CPacketPlayerDigging.Action.START_DESTROY_BLOCK, pos, mc.objectMouseOver.sideHit))
+        } else {
+            mc.connection!!.sendPacket(CPacketPlayerDigging(CPacketPlayerDigging.Action.STOP_DESTROY_BLOCK, pos, mc.objectMouseOver.sideHit))
+        }
+        mc.player.swingArm(EnumHand.MAIN_HAND)
+    }
+
+    private fun placeBlock(pos: BlockPos, mat: Block): Boolean
+    {
```

Comment:
use the same line for {

Reply:
i normally do so it has to come from autofeetplace xD